django-stubs/contrib/gis/db/models/functions.pyi

```diff
@@ -1,5 +1,7 @@
 from typing import Any
 
+from django.db.backends.base.base import BaseDatabaseWrapper
+from django.db.backends.mysql.compiler import SQLCompiler
```

Comment:
```suggestion
from django.db.models.sql.compiler import SQLCompiler, _AsSqlType
```

Reply:
Thanks

Reply:
Looks like this has regressed again...

Reply:
Ah I get it now, I did a rebase+force push without pulling the commit added by GitHub "Commit suggestion". My bad